recipe/meta.yaml

```diff
@@ -14,20 +14,23 @@ source:
     - 0001-common-ompio-fix-calculation-in-simple-grouping-opti.patch
 
 build:
-  number: 2
-  skip: True  # [win]
+  number: 3
+  skip: True  # [win or (linux64 and cuda_compiler_version != '9.2')]
 
 outputs:
   - name: openmpi
     script: build-mpi.sh
     build:
       run_exports:
         - {{ pin_subpackage('openmpi', min_pin='x.x.x', max_pin='x.x') }}
+      ignore_run_exports:
+        - cudatoolkit
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('fortran') }}
+        - {{ compiler('cuda') }}  # [linux64]
         - make  # [unix]
         - perl 5.26.2
       host:
```

Comment:
Would add the following after `run`. This should make sure that users are not able to install a `cudatoolkit` older than the one we built with.

```yaml
      run_constrained:
        - cudatoolkit >={{ cuda_compiler_version }}
```

Reply:
Thanks, John! Added in aea185b.

Reply:
cc @isuruf (in case he has a better suggestion on how to add this 😉)

Reply:
Does this really work with any CUDA version if built using 9.2?

Reply:
Yeah that's what we found through experimentation and discussion with OpenMPI devs. Would read [this thread]( https://github.com/conda-forge/openmpi-feedstock/pull/54#discussion_r369764497 ) and [this thread]( https://github.com/conda-forge/openmpi-feedstock/issues/42#issuecomment-577813078 ) for more info.

Reply:
Thanks @jakirkham!